server/src/controllers/__tests__/user.spec.ts

```diff
@@ -63,18 +63,18 @@ describe("UserController", () => {
   })
 
   it("should create a user", async () => {
-    const mockUser: [{ id: number; email: string }, boolean] = [{ id: 1, email: "test@test.com" }, true]
-    const mockScore: [{ userId: number; score: number }, boolean] = [{ userId: 1, score: 0 }, true]
-    const expectedFullUser = { user: mockUser[0], scores: mockScore[0] }
+    const mockUser: [{ id: number; email: string; streak: 0 }, boolean] = [
+      { id: 1, email: "test@test.com", streak: 0 },
+      true,
+    ]
+    const expectedUser = mockUser[0]
 
     ;(database.User.findOrCreate as jest.Mock).mockResolvedValue(mockUser)
-    ;(database.Score.findOrCreate as jest.Mock).mockResolvedValue(mockScore)
 
     const fullUser = await UserController.createUser("test@test.com")
 
     expect(database.User.findOrCreate).toHaveBeenCalledWith({ where: { email: "test@test.com" } })
-    expect(database.Score.findOrCreate).toHaveBeenCalledWith({ where: { userId: mockUser[0].id } })
-    expect(fullUser).toEqual(expectedFullUser)
+    expect(fullUser).toEqual(expectedUser)
```

Comment:
Doesn't return a FullUser anymore because we don't need scores when just creating a user.